virttest/libvirt_remote.py

```diff
@@ -44,3 +44,20 @@ def update_remote_file(params, value,
     except Exception as err:
         raise exceptions.TestFail("Unable to update {}: {}"
                                   .format(file_path, err))
+
+
+def delete_remote_file(file, params):
+    """
+    Delete file on remote
+
+    :param file: The file to be deleted
+    :param params: Test params
+    """
+    remote_ip = params.get("server_ip", params.get("remote_ip"))
+    remote_pwd = params.get("server_pwd", params.get("remote_pwd"))
+    remote_user = params.get("server_user", params.get("remote_user"))
```

Comment:
Do we need check validity of those parameters?

Reply:
updated.